connect/src/model.rs

```diff
@@ -23,19 +23,26 @@ pub enum PlayingTrack {
     Uid(String),
 }
 
-impl From<SkipTo> for PlayingTrack {
-    fn from(value: SkipTo) -> Self {
-        // order of checks is important, as the index can be 0, but still has an uid or uri provided,
-        // so we only use the index as last resort
-        if let Some(uri) = value.track_uri {
-            PlayingTrack::Uri(uri)
-        } else if let Some(uid) = value.track_uid {
-            PlayingTrack::Uid(uid)
-        } else {
-            PlayingTrack::Index(value.track_index.unwrap_or_else(|| {
-                warn!("SkipTo didn't provided any point to skip to, falling back to index 0");
-                0
-            }))
+impl From<Option<SkipTo>> for PlayingTrack {
```

Comment:
Can you instead of adjusting the `From`, derive `Default` for `SkipTo` and `unwrap_or_default`?

Reply:
Thanks, that's nicer.

Reply:
Yeah, you can easily run into deep indentation issues with rust. There are ways around it, but usually they are not the obvious one. Anyhow thanks for the adjustment^^